tests/e2e/e2e_egress_validator_test.go

```diff
@@ -0,0 +1,13 @@
+package e2e
```

Comment:
nit: whitespace looks off to me.

Reply:
removed the blank line if that's what you were referring to

Reply:
I was referring to the indentation. It looks like more than 4 spaces?